ledger/totals_test.go

```diff
@@ -69,7 +68,7 @@ func TestAlgoCountMarshalMsg(t *testing.T) {
 	inBuffer := make([]byte, 0, 128)
 	outBuffer, err := ac.MarshalMsg(inBuffer)
 	require.NoError(t, err)
-	fmt.Printf("out %d in %d\n", len(outBuffer), len(inBuffer))
+	//fmt.Printf("out %d in %d\n", len(outBuffer), len(inBuffer))
```

Comment:
forgotten debug output?

Reply:
We don't really need this debug output, unless there is a problem. I don't think it's very valuable at this time, and if we'll find that the subsequent entry fail, we could improve it by making it into a require.Truef(...) which would include the above information.

Reply:
I meant to delete it.
In logic tests Brian established a pattern like
```
if err != nil {
// print trace
}
require.NoError(t, err)
```

Reply:
updated.